File: src/com/wang/Game2dEngine/painter/interfaces/IUserInteraction.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.wang.Game2dEngine.painter.interfaces;

import java.awt.event.ComponentAdapter;
import java.awt.event.MouseListener;

/**
 *
 * @author ricolwang
 */
public interface IUserInteraction
{

    public void addMouseListener(MouseListener listener);

    public void addComponentListener(ComponentAdapter componentAdapter);
}

File: src/com/wang/Game2dEngine/painter/interfaces/IUserInteraction.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.wang.Game2dEngine.painter.interfaces

import java.awt.event.ComponentAdapter
import java.awt.event.MouseListener

/**
 * @author ricolwang
 */
interface IUserInteraction
{
    fun addMouseListener(listener: MouseListener?)
    fun addComponentListener(componentAdapter: ComponentAdapter?)
}